platforms/Platform_Definitions.hpp

```diff
@@ -0,0 +1,12 @@
+#include <SoundSystem.hpp>
+
+//				Sound Systems
+//-----------------------------------------
+
+// Windows Sound System (Direct Sound)
+#if 1
```

Comment:
Use platform specific defines, for example, `#ifdef _WIN32`, `#elif defined(USE_SDL)` etc.

Reply:
I think the issue with using platform specific platforms like _WIN32 or otherwise is that you might not want to use a platform specific code and use one that is cross-platform, for example I am making an OpenAL implementation and I might want to use that over the windows specific sound system or vice versa

Reply:
Okay, sure, have the OpenAL take over if USE_OPENAL is enabled for instance